lib/http_clients/creditas/person_deletion_api.ex

```diff
@@ -1,6 +1,26 @@
 defmodule HttpClients.Creditas.PersonDeletionApi do
   @moduledoc false
 
+  alias HttpClients.Creditas.PersonDeletionApi.PersonDeletion
+
+  @spec get(Tesla.Client.t(), binary()) :: {:error, any}
+  def get(client, person_deletion_id) do
+    case Tesla.get(client, "/person-deletions/#{person_deletion_id}") do
+      {:ok, %Tesla.Env{status: 200, body: body}} -> {:ok, build_person_deletion(body)}
```

Comment:
não deveria ser um 
```suggestion
      {:ok, %Tesla.Env{status: 200, body: %{"data" => data}}} -> {:ok, build_person_deletion(data)}
```
ou retorna no body direto mesmo?

Reply:
segundo a [doc](https://developers.creditas.io/reference/persondeletion#searchpersondeletionbyid), não